src/gather_stats.ts

```diff
@@ -493,7 +574,11 @@ async function async_main() {
 
         console.log(`Time: ${script_finish.diff(script_start).as('seconds').toFixed(3)}s`);
     } catch (error) {
-        console.log(`ERROR: ${error.message}`);
+        if (error instanceof Error) {
+            console.log(`ERROR: ${error.message}`);
+        } else {
+            console.log(`UNKNOWN ERROR: ${error}`);

```

Comment:
Don't you mean "UNKNOWN non-`Error`"? [No change requested. Just enjoying the irony of the "If the `error` is not an `Error`" branch.]